charts/istioconfig/templates/request-header-checker.yaml

```diff
@@ -0,0 +1,24 @@
+# Define a rule to reject all request if oidc_access_token is presented in request header.
```

Comment:
Are the configurations in this file specific to a namespace or global? Any configuration not specific to a namespace/environment, should instead go in the terra-cluster-networking chart.

Reply:
resolved offline